src/Subscriber/Mock.php

```diff
@@ -60,6 +60,12 @@ public function onBefore(BeforeEvent $event)
             }
         }
 
+        $save_to = $event->getRequest()->getConfig()->get('save_to');
+
+        if ($save_to) {
+            file_put_contents($save_to, $item->getBody());
```

Comment:
save_to can be of various types: http://docs.guzzlephp.org/en/latest/clients.html#save-to. I think this method would need to account for that.